ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/ImageVulnerabilitiesTable.tsx

```diff
@@ -302,6 +310,12 @@ function ImageVulnerabilitiesTable({
                                     >
                                         <DateDistance date={discoveredAtImage} />
                                     </Td>
+                                    <Td
```

Comment:
How about `modifier="nowrap"` prop as in DeploymentVulnerabilitiesTable.tsx file?

Picture at 1380px width:
![ImageVulnerabilitiesTable_nowrap_1380px](https://github.com/user-attachments/assets/2ba79884-a5af-4663-8841-4f4688de907d)